setup.py

```diff
@@ -21,9 +21,6 @@ def package_data(pkg, root):
     packages=[
         'recommender',
     ],
-    install_requires=[
-        'XBlock',
```

Comment:
This was removed since it the hosting platform (xblock-sdk or edx-platform) provides XBlock already, and at a specified version. We want the install to come from there. Adding our own version here, in some cases, cause XBlock to be installed at an incompatible version, breaking the host.